gateleen-routing/src/main/java/org/swisspush/gateleen/routing/Rule.java

```diff
@@ -182,9 +184,11 @@ public String getStorage() {
         return storage;
     }
 
-    public void setStorage(String storage) {
-        this.storage = storage;
-    }
+    public void setStorage(String storage) { this.storage = storage; }
+
+    public ProxyOptions getProxyOptions() { return proxyOptions; }
```

Comment:
code format - this is just a one-liner. Weird.
(although IDEs typically present such one-liner-methods in this compact form it should not be a one-liner in the Sourcecode-File itself)